libc/utils/gpu/server/rpc_server.cpp

```diff
@@ -167,8 +167,9 @@ static void handle_printf(rpc::Server::Port &port, TempStorage &temp_storage) {
       continue;
 
     char *buffer = temp_storage.alloc(buffer_size[lane]);
-    WriteBuffer wb(buffer, buffer_size[lane]);
-    Writer writer(&wb);
+    WriteBuffer<WriteMode::FILL_BUFF_AND_DROP_OVERFLOW> wb(buffer,
```

Comment:
If this enum is the most common, should we make it the default? Or is it better to be explicit every time?